config/jobs/kubernetes/sig-testing/verify.yaml

```
@@ -207,5 +208,12 @@ periodics:
       - name: REPO_DIR
         value: /workspace/k8s.io/kubernetes
       # docker-in-docker needs privileged mode
+      resources:
+        limits:
+          cpu: 6
+          memory: 24Gi
+        requests:
+          cpu: 6
+          memory: 24Gi
       securityContext:
         privileged: true
```

I'll suggest to start small and increase resources consumption if needed.
we can now monitor those jobs in https://monitoring-eks.prow.k8s.io/?orgId=1

```suggestion
      resources:
        limits:
          cpu: 2
          memory: 4Gi
        requests:
          cpu: 2
          memory: 4Gi
      # docker-in-docker needs privileged mode
      securityContext:
        privileged: true
```